gui/wxpython/mapdisp/preferences.py

```diff
@@ -0,0 +1,329 @@
+"""
+@package mapdisp.settings

```

Comment:
I just noticed the `settings` here but the file is named `preferences.py`. I would perhaps go with `settings` here (not strong opinion), so please rename the class as well.

Reply:
Well, I changed it to preferences.py because in gui_core it is also called preferences.py. But anyway, settings.py could be anything (very general), I think preferences.py more points on the purpose. I would let it as it is, and change just the @package name if it okey. :-)

Reply:
I guess my point was to avoid preferences, because that's used for something else, this is not actually setting any preferences, you changing the actual behavior for specific mp display. Settings is used elsewhere, so that may not be the right choice either. How about rename this to properties.py and use *property* for naming of the classes, because that's the terminology we use for this already?

Reply:
Right, I think properties are better than preferences. Good idea! :-)